src/com/palmergames/bukkit/towny/utils/TownPeacefulnessUtil.java

```diff
@@ -89,7 +97,7 @@ public static void punishPeacefulPlayersInActiveSiegeZones() {
 					continue;
 
 				//Dont apply if player has the immunity perm
-				if (TownyUniverse.getInstance().getPermissionSource().testPermission(player, PermissionNodes.TOWNY_SIEGE_WAR_IMMUNE_TO_WAR_NAUSEA.getNode()))
+				if (TownyUniverse.getInstance().getPermissionSource().has(player, SiegeWarPermissionNodes.TOWNY_SIEGE_WAR_IMMUNE_TO_WAR_NAUSEA.getNode()))
```

Comment:
I haven't taken a close enough look at the TownPeacefullnessUtil to see what use it will have in the future. For now it is SW exclusive.

Reply:
Since the [force push] messed up the links, I've recommenced on the new version of the code ([here]) for reference.

[force push]: https://github.com/TownyAdvanced/Towny/compare/6cec7955c7b06dcf3df81d3cf249a2f1aac0c66c..8ff4c358df50cb3e830c397645ebf322db484380
[here]: https://github.com/TownyAdvanced/Towny/pull/4472#discussion_r531166184

Reply:
> I haven't taken a close enough look at the TownPeacefullnessUtil to see what use it will have in the future. For now it is SW exclusive.

Fair enough.

Reply:
This is a SW node, so the code that uses it will likely end up apart from the TownPeacefullnessUtil (if the util stays in the towny/utils package at all.